podpac/core/data/datasource.py

```diff
@@ -398,12 +397,13 @@ def eval(self, coordinates, output=None):
         # return the output to the originally requested output dims
         if requested_dims is not None and requested_dims != output.dims:
             output = output.transpose(*requested_dims)
-        
-        # transform the output crs to the requested coordinates crs
-        # TODO: this could be made into method in/for UnitsDataArray 
+
+        # if requested crs is differented than native coordinates,
+        # fabricate a new output with the original coordinates and new values
         if self._evaluated_coordinates.crs != coordinates.crs:
-            t_coords = coordinates.transform(self._evaluated_coordinates.crs)
-            output = output.assign_coords(**t_coords.coords)
+            trans_output = self.create_output_array(self._evaluated_coordinates)
+            trans_output[:] = output[:].values
```

Comment:
@jmilloy @mpu-creare I decided to replace the output coordinates with the originally requested coordinates if the crs has to be changed. This avoids issues when returning Dependent Coordinates